backends/capabilities_android/java/org/haxe/duell/capabilities/Capabilities.java

```diff
@@ -72,6 +75,22 @@ public static double getDensity()
         return metrics.density;
     }
 
+    public static double getTotalMemory()
+    {
+        MemoryInfo mi = new MemoryInfo();
+        ActivityManager activityManager = (ActivityManager) DuellActivity.getInstance().getSystemService(Context.ACTIVITY_SERVICE);
+        activityManager.getMemoryInfo(mi);
+
+        if (android.os.Build.VERSION.SDK_INT > android.os.Build.VERSION_CODES.JELLY_BEAN)
```

Comment:
This means that for pretty much 99% of the devices out there which are targeted, this returns 0. Doesn't look useful.